src/cmd/ign.cc

```diff
@@ -15,9 +15,9 @@
  *
 */
 #include <cstring>
-#include <ignition/common/Console.hh>
+#include <gz/common/Console.hh>
 
-#include "ignition/launch/config.hh"
+#include "gz/launch/config.hh"
```

Comment:
I think you'll need to update this line too

https://github.com/gazebosim/gz-launch/blob/8c78b6c937943ba1650e887c3a82dbfaea6c154f/src/cmd/ign.cc#L33